torch/_prims/executor.py

```diff
@@ -45,34 +45,52 @@ def call_function(self, target, args, kwargs):
                     args = tuple(map(_to_nvfuser_constant, args))
```

Comment:
This change is required and all other points are just for discussion and shouldn't hold this PR from being merged if tests pass.

Reply:
IIRC, when I wrote this code, there was no such `_to_nvfuser_constant`, so seems that we did the same work in different ways.

Reply:
I actually need to keep the `_to_nvfuser_constant` instead of using the `to_nv`, because according to https://github.com/pytorch/pytorch/pull/78655#discussion_r899858111, `to_nv` will convert numbers as scalar, and we will need additional inputs to the fusion.